src/components/now-playing-drawer/PlayBar.tsx

```diff
@@ -223,13 +218,13 @@ export const PlayBar = ({ onPress, opacityAnim }: PlayBarProps) => {
           </View>
           <View style={styles.trackText}>
             <Text numberOfLines={1} weight='bold' style={styles.title}>
-              {trackInfo?.title ?? ''}
+              {track?.title ?? ''}
             </Text>
             <Text weight='bold' style={styles.separator}>
```

Comment:
it might be nice to start adding some a11y hidden roles for things that are presentational, like separators? https://reactnative.dev/docs/accessibility#accessibilityelementshidden-ios